### DISCUSSION TITLE
Spelling Error on Website

### DISCUSSION THREAD

**Discussion by oOStroudyOo** [tteck/Proxmox#980] [Help Needed] [answered]
Hi Tteck,

I wasn't sure where to post this but the Transmission LXC script on your website wouldn't run.
I tried it without the upper case T and it works, you'll just need to make that small adjustment on your website.

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/transmission-v5.sh)"

Thank you so much for all of your amazing work.

**Comment by tteck** [comment, marked as answer]
Oops... thank you! FIXED
